apps/api/src/app/events/usecases/process-subscriber/process-subscriber.usecase.ts

```diff
@@ -90,10 +95,21 @@ export class ProcessSubscriber {
         firstName: subscriberPayload.firstName,
         lastName: subscriberPayload.lastName,
         phone: subscriberPayload.phone,
+        avatar: subscriberPayload.avatar,
       })
     );
   }
 
+  private subscriberNeedUpdate(subscriber: SubscriberEntity, subscriberPayload: ISubscribersDefine): boolean {
+    return (
+      subscriber.email !== subscriberPayload.email ||
```

Comment:
```suggestion
      subscriberPayload.email && subscriber.email !== subscriberPayload.email ||
```

@djabarovgeorge I'm thinking about adding this to each clause, this will make sure that if during the trigger event a smaller scope of the subscriber was passed than the one that exists on the subscriber entity. In this case without this check it will still always perform an update call.

Reply:
oh my, I missed that one! 
great thinking I will add it asap!

@scopsy thanks for noticing!